test/helpers/data/staging-workload-helper.js

```diff
@@ -50,28 +50,67 @@ const aliases = {
   custtierc1: 'custtierc1',
   custtierb2: 'custtierb2',
   custtierb1: 'custtierb1',
-  custtiera: 'custtiera'
+  custtiera: 'custtiera',
+  assessmentDate: 'assessment_date',
+  assessmentCode: 'assessment_code',
+  assessmentDescription: 'assessment_desc',
+  assessmentStaffName: 'assessment_staff_name',
+  assessmentStaffKey: 'assessment_staff_key',
+  assessmentStaffGrade: 'assessment_staff_grade',
+  assessmentTeamKey: 'assessmentent_team_key',
+  assessmentProviderCode: 'assessment_provider_code',
+  crn: 'crn',
+  disposalOrReleaseDate: 'disposal_or_release_date',
+  sentenceType: 'sentence_type',
+  soRegistrationDate: 'so_registration_date'
 }
 
 const wmtExtractTable = 'wmt_extract'
 const courtReportsTable = 'court_reports'
 const institutionalReportsTable = 'inst_reports'
+const armsTable = 'arms'
 
 const testOmKey = 'l102A'
+const testTeamCode = 'KNS'
 // Create a test CaseDetails object, mapping properties to column names
 
 module.exports.getTestCaseSummary = function (omKey = testOmKey) {
   return stagingHelper.getTestCaseSummary(omKey)
 }
 
-module.exports.getTestCourtReport = function (omKey = testOmKey) {
-  return stagingHelper.getTestCourtReport(omKey)
+module.exports.getTestCourtReport = function (omKey = testOmKey, teamCode = testTeamCode) {
+  return stagingHelper.getTestCourtReport(omKey, undefined, teamCode)
 }
 
 module.exports.getTestInstitutionalReport = function (omKey = testOmKey) {
   return stagingHelper.getTestInstitutionalReport(omKey)
 }
 
+module.exports.getArmsData = function (omKey = testOmKey, teamCode = testTeamCode) {
+  var defaultArmsObject = {
+    assessmentDate: '09/10/2017',
```

Comment:
Are we always defaulting to specific assessmentDate or is this to be changed?

Reply:
There is no date querying on the application side, this is just a test object to ensure we can correctly retrieve the property from the DB so the actual value will have no effect.

Reply:
Yes, its the test helper 👍